projects/mono.md

```diff
@@ -0,0 +1,31 @@
+# Mono
+
+Mono, the open source development platform based on the .NET Framework, allows developers to build cross-platform applications with improved developer productivity. Mono’s .NET implementation is based on the ECMA standards for [C#](http://www.ecma-international.org/publications/standards/Ecma-334.htm) and the [Common Language Infrastructure](http://www.ecma-international.org/publications/standards/Ecma-335.htm).
```

Comment:
Does it make sense to talk about ECMA 334, since that's currently at C# 2.0 (and C# 5.0 some time in the future), while the Mono C# compiler supports C# 6.0?